Assets/MixedRealityToolkit.Providers/WindowsMixedReality/XR2018/WindowsMixedRealityEyeGazeDataProvider.cs

```diff
@@ -249,7 +256,6 @@ private Ray SmoothGaze(Ray? newGaze)
                 smoothedGaze.direction = newGaze.Value.direction;
                 smoothedGaze.origin = newGaze.Value.origin;
                 confidenceOfSaccade = 0;
-                isSaccading = false;
```

Comment:
This value isn't used after it's set to `false` here, so I removed the set to clarify that it doesn't matter.